ui/hooks/useIsOriginalNativeTokenSymbol.js

```diff
@@ -1,15 +1,34 @@
 import { useEffect, useState } from 'react';
 import { useSelector } from 'react-redux';
 import fetchWithCache from '../../shared/lib/fetch-with-cache';
-import { CHAIN_ID_TO_CURRENCY_SYMBOL_MAP } from '../../shared/constants/network';
+import {
+  CHAIN_ID_TO_CURRENCY_SYMBOL_MAP,
+  CHAIN_ID_TO_CURRENCY_SYMBOL_MAP_NETWORK_COLLISION,
+} from '../../shared/constants/network';
 import { DAY } from '../../shared/constants/time';
 import { useSafeChainsListValidationSelector } from '../selectors';
+import { getValidUrl } from '../../app/scripts/lib/util';
 
-export function useIsOriginalNativeTokenSymbol(chainId, ticker, type) {
+export function useIsOriginalNativeTokenSymbol(
+  chainId,
+  ticker,
+  type,
+  rpcUrl = null,
+) {
   const [isOriginalNativeSymbol, setIsOriginalNativeSymbol] = useState(null);
   const useSafeChainsListValidation = useSelector(
     useSafeChainsListValidationSelector,
   );
+
+  const isLocalhost = (urlString) => {
```

Comment:
Could we use the existing `isLocalhostOrHttps` here?

Reply:
Yes